index.html

```diff
@@ -0,0 +1,378 @@
+                        <li><img class="social-logo" src="/images/baseline_facebook_black_18dp.png" alt="FB logo"></li>
+                        <li><img class="social-logo" src="/images/baseline_facebook_black_18dp.png" alt="FB logo"></li>
+                        <li><img class="social-logo" src="/images/baseline_facebook_black_18dp.png" alt="FB logo"></li>
+                        <li><img class="social-logo" src="/images/baseline_facebook_black_18dp.png" alt="FB logo"></li>
+                        <li><img class="social-logo" src="/images/baseline_facebook_black_18dp.png" alt="FB logo"></li>
+                    </ul>
+                </div>
+            </div>
+            
+            <figure>
+                <img src="/images/18cosmos-superJumbo.jpg" alt="Alan Guth was one of the first physicists to hypothesize the existence of inflation, which explains how the universe expanded so uniformly and so quickly in the instant after the Big Bang 13.8 billion years ago.Credit...">
+                <div class="guth-text">
+                    <span>Alan Guth was one of the first physicists to hypothesize the existence of inflation, which explains how the universe expanded so uniformly and so quickly in the instant after the Big Bang 13.8 billion years ago.</span>
+                    <span>Rick Friedman for The New York Times</span>
+                </div>
+            </figure>
+            
+        </section>
+    </header>
+    
+    <main>
+        <div>
+            <section class="text-title">
+                <h4>By Dennis Overbye</h4>
+                <p>March 17, 2014</p>
+            </section>
+            <section class="text">
+                <p>CAMBRIDGE, Mass. — One night late in 1979, an itinerant young physicist named Alan Guth, with a new son and a year’s appointment at Stanford, stayed up late with his notebook and equations, venturing far beyond the world of known physics.</p>
+                <p>He was trying to understand why there was no trace of some exotic particles that should have been created in the Big Bang. Instead he discovered what might have made the universe bang to begin with. A potential hitch in the presumed course of cosmic evolution could have infused space itself with a special energy that exerted a repulsive force, causing the universe to swell faster than the speed of light for a prodigiously violent instant.</p>
+                <p>If true, the rapid engorgement would solve paradoxes like why the heavens look uniform from pole to pole and not like a jagged, warped mess. The enormous ballooning would iron out all the wrinkles and irregularities. Those particles were not missing, but would be diluted beyond detection, like spit in the ocean.</p>
+                <p>“SPECTACULAR REALIZATION,” Dr. Guth wrote across the top of the page and drew a double box around it.</p>
+
+                
+
+            </section>
+
+            <aside>
+                <h4>Editors’ Picks</h4>
```

Comment:
This section is extending more than it should in to the right side which is making your site to expand more than it should, kindly fix this.